test/e2e/specs/realrp/realrp.go

```diff
@@ -28,8 +28,9 @@ var _ = Describe("Resource provider e2e tests [Default][Real]", func() {
 	// NOTE: Ensure this is always the first test in the [Default][Real] spec!
 	It("should deploy a cluster using the production RP", func() {
 		var err error
-		cli, err = azure.NewClientFromEnvironment(false)
+		cli, err = azure.NewClientFromEnvironment(false, true)
```

Comment:
how come this isn't using RealRPClient?

Reply:
I wanted to be 100% sure FakeRP e2e work first. Now it uses RealRPClient.